muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/components/guider/body/application/current-student/workspaces.tsx

```diff
@@ -20,12 +21,14 @@ class CurrentStudentWorkspaces extends React.Component<CurrentStudentWorkspacesP
   render(){
     //TODO WAT?... mf???? o.o communicator?...
```

Comment:
Please remove these comments as they are not even remotely written in professional manner.